tests/kafkatest/services/kafka/kafka.py

```diff
@@ -503,7 +503,7 @@ def create_topic(self, topic_cfg, node=None, use_zk_to_create_topic=True):
         self.logger.info("Running topic creation command...\n%s" % cmd)
         node.account.ssh(cmd)
 
-    def delete_topic(self, topic, node=None):
+    def delete_topic(self, topic, node=None, use_zk_to_delete_topic=False):
```

Comment:
create/describe/list use `use_zk_to_..._topic=True` so I'm wondering if it is best to remain consistent here.  For example, if running an older version, might `--bootstrap-server` not be available?

Reply:
Yeah that's a good point, but in this scenario I was thinking to remain consistent with the original implementation. Also I'm not sure how far we want to backport this, but if we backport it past the version where `--bootstrap-server` is not available, then I think we would have to rewrite this function to account for that. Also since we are on our way to deprecate and remove ZK, this would be a good way to start pushing us towards doing that.

Reply:
@rondagostino are we ok with merging this to trunk? Since this is not required for existing tests which either use ZK or PLAINTEXT brokers, not planning to backport to older versions.

Reply:
@rajinisivaram Yes, merging to just trunk seems fine to me.